src/pystatis/table.py

```diff
@@ -104,88 +125,103 @@ def get_data(
         self.metadata = metadata
 
     @staticmethod
-    def prettify_table(data: pd.DataFrame, db_name: str) -> pd.DataFrame:
+    def prettify_table(data: pd.DataFrame, db_name: str, language: str) -> pd.DataFrame:
         """Reformat the data into a more readable table
 
         Args:
             data (pd.DataFrame): A pandas dataframe created from raw_data
             db_name (str): The name of the database.
+            language (str): The requested language. One of "de" or "en".
 
         Returns:
             pd.DataFrame: Formatted dataframe that omits all unnecessary Code columns
             and includes informative columns names
         """
         match db_name:
             case "genesis":
-                pretty_data = Table.parse_genesis_table(data)
+                pretty_data = Table.parse_genesis_table(data, language)
```

Comment:
If `language` is used throughout the class, maybe we should add it as parameter to `__init__` and make a proper class instance variable out of it? Default can still be "de"

Reply:
I am not sure about this. In general, I would favor treating the language just like any other query parameter and pass it to get_data.

Reply:
However, it might make the code easier if we treat it as a class instance variable...

Reply:
I don't think many people will use "en" anyway so let's keep it simple right now and everything as parameter of get_data so we can provide a single interface